scripts/tox_install_cmd.sh

```diff
@@ -13,6 +13,9 @@ done
 
 echo "${pip_options[*]}"
 
+# Default to empty, to ensure snowflake_path variable is defined.
+snowflake_path=${snowflake_path:-""}
```

Comment:
why we need this?

Reply:
I got an error exit code 2, unbound variables. This fixes it.